lib/connection.ts

```diff
@@ -494,6 +537,19 @@ export class Connection extends Entity {
     return session.createSender(options);
   }
 
+  /**
+   * Creates an asynchronous amqp sender. It either uses the provided session or creates a new one.
+   * @param {SenderOptionsWithSession} options Optional parameters to create a sender link.
+   * @return {Promise<Sender>} Promise<Sender>.
+   */
+  async createAsynchronousSender(options?: AsynchronousSenderOptionsWithSession): Promise<AsynchronousSender> {
+    if (options && options.session && options.session.createSender) {
```

Comment:
Shouldnt the check for `options.session.createSender` here be for `options.session.createAsynchronousSender` instead?

Reply:
Yup. Will update this as well.